src/operations/blas3/gemm_no_local.hpp

```diff
@@ -188,20 +196,36 @@ class Gemm<input_t, output_t, DoubleBuffer, NbcA, NbcB, ClSize, tile_type,
     const index_t tile_id_row = wg_id % number_of_block_per_row;
     /* column tile id per work group */
     const index_t tile_id_col = wg_id / number_of_block_per_row;
-    /* work item id per row */
-    const index_t local_item_id_row = item_id % wg_rows;
-    /* work item id per column */
-    const index_t local_item_id_col = item_id / wg_rows;
     /* the start position of the tile-row per work group */
     const index_t wg_row = tile_id_row * block_rows;
     /* the start position of the tile-column per work group */
     const index_t wg_col = tile_id_col * block_cols;
+    /*!
+     * @brief is_internal_block_m and is_internal_block_n is used to distinguish
+     * the internal block. Therefore, work items using these blocks dont need to
+     * check for boundaries.
+     */
+    const bool is_internal_block =
+        (m - wg_row >= block_rows) && (n - wg_col >= block_cols);
+
+    /* @brief Packet sizes for A and B. A can only be vectorised if it is not
```

Comment:
For Doxygen you need to start the comment with `/**` or `/*!`. But even then this would only apply to `a_packet_size`. Might be nicer to do `///` instead for variables.

Also `vectorised`.

Reply:
Fixed, thanks!